config/config.js

```diff
@@ -115,7 +115,9 @@ var initGlobalConfig = function () {
   initGlobalConfigFiles(config, assets);
 
   // Run in HTTP and let the proxy handle SSL termination
-  config.secure.ssl = false;
+  if (config.secure) {
```

Comment:
Needed this when `NODE_ENV` is `development` because the development config doesn't have a "secure" section.